src/lib/Warn.svelte

```diff
@@ -0,0 +1,69 @@
+<script lang="ts">
+    /**
```

Comment:
I think we should rename this component to `Alert` and pass a `variant` prop as in the `Chip` component. Then, when we do `<Alert variant="warning" />` (or `danger`) we get the original `Warn` component. This way the component could be used to display successful or info events as well

Reply:
@acuarica actually, I've been working on the `Alert` component and I think it'll be way easier and more robust to simply use a component library and same applies for the images. Have a look at the following resources:
https://project-awesome.org/TheComputerM/awesome-svelte

Reply:
Ok, here's my list of UI kits in order of preference

1. https://carbon-components-svelte.onrender.com/
2. https://sveltematerialui.com/
3. https://smeltejs.com/